zng/cc.rs

```diff
@@ -11,7 +11,17 @@ struct Build {
 
 impl Build {
     fn new(cfg: cc::Build) -> Self {
-        let is_msvc = cfg.get_compiler().is_like_msvc();
+        // cc currently has a bug where they create a named temp file in a directory
+        // without ensuring the directory exists first, so apply this workaround
+        // until it can be fixed upstream
```

Comment:
```suggestion
        // until it can be fixed upstream
        // Can be removed once https://github.com/rust-lang/cc-rs/pull/1072 is merged and released.
```

Reply:
Oh, turns out that I can't commit my own suggestions. Maybe you can commit it so I can merge?